src/Microsoft.Extensions.Localization/ResourceManagerStringLocalizerFactory.cs

```diff
@@ -50,6 +50,29 @@ public class ResourceManagerStringLocalizerFactory : IStringLocalizerFactory
             }
         }
 
+        protected virtual string GetKey(TypeInfo typeInfo)
```

Comment:
`ResourceManager` calls it "baseName".  I think that's a little obscure in our use case, maybe "GetResourceBaseName"? 100% agree now that you've pointed it out that "Key" is a terrible noun here.